book/src/generalized-ownership.md

```diff
@@ -0,0 +1,279 @@
+# Generalized Ownership
+
+This design describes generalized ownership of countable items on the Solana
+blockchain. The goal of this design is to propose a single general-purpose
+program whose state can be used across many different programs to implement
+ownership, transfer and programmatic assignment. 
+
+Many programs in Solana should be able to use a common program similar to
```

Comment:
"use a common program" -> "define an instruction"?

Reply:
@garious not really define, how about: "reuse the same code and state machine"